neurom/__init__.py

```diff
@@ -59,13 +59,15 @@
 '''
 
 import logging as _logging
-from .version import VERSION as __version__
-from .core import iter_neurites, iter_sections, graft_neuron, iter_segments, NeuriteType
+
+from .core import (NeuriteType, graft_neuron, iter_neurites, iter_sections,
+                   iter_segments)
+from .core._neuron import Neuron
```

Comment:
Why not include it in the upper import? In `core/__init__.py` we have `from ._neuron import (Section, Neurite, Neuron, `

Reply:
That's a good question. I actually don't know what is the proper Python standard policy but I thought since Neuron is really a base object of NeuroM it deserved to be avalailable from the top level.

Reply:
So, can we use a single import statement with `Neuron` included?:
```
from .core import (NeuriteType, Neuron, graft_neuron, iter_neurites, iter_sections,
                   iter_segments)
```